### ISSUE TITLE
component create should fail before if directory does not exist

### ISSUE THREAD

**Issue by containscafeine** [redhat-developer/odo#52]
```console
$ ocdev component create nodejs --dir ./nodejs-ex                                                      
--> Found image cbb7eed (5 days old) in image stream "openshift/nodejs" under tag "6" for "nodejs"                             

    Node.js 6                  
    ---------                  
    Node.js 6 available as docker container is a base platform for building and running various Node.js 6 applications and frameworks. Node.js is a platform built on Chrome's JavaScript runtime for easily building fast, scalable network applications. Node.js uses an event-driven, non-blocking I/O model that makes it lightweight and efficient, perfect for data-intensive real-time applications that run across distributed devices.

    Tags: builder, nodejs, nodejs6                             

    * A source build using binary input will be created        
      * The resulting image will be pushed to image stream "nodejs:latest"                                                     
      * A binary build was created, use 'start-build --from-dir' to trigger a new build                                        
    * This image will be deployed in deployment config "nodejs"                                                                
    * Port 8080/tcp will be load balanced by service "nodejs"  
      * Other containers can access this service through the hostname "nodejs"                                                 

--> Creating resources with label app.ocdev.developers.redhat.com=app,component.ocdev.developers.redhat.com=nodejs ...         
    imagestream "nodejs" created                               
    buildconfig "nodejs" created                               
    deploymentconfig "nodejs" created                          
    service "nodejs" created   
--> Success                    
    Build scheduled, use 'oc logs -f bc/nodejs' to track its progress.                                                         
    Application is not exposed. You can expose services to the outside world by executing one or more of the commands below:   
     'oc expose svc/nodejs'    
    Run 'oc status' to view your app.                          

please wait, building application...                           
failed to execute oc command   
 error: stat /home/concaf/ocdev/src/github.com/redhat-developer/ocdev/nodejs-ex: no such file or directory                    
```

**Comment by kadel** [comment]
related to https://github.com/redhat-developer/ocdev/issues/192

**Comment by cdrage** [comment]
Yeah, looks like this has yet to be implemented, I'll assign this to myself.

```sh
▶ odo create nodejs foobar --local alksdjflasjdfkljsakdljl
Component 'foobar' was created.
To push source code to the component run 'odo push'

Component 'foobar' is now set as active component.
```

**Comment by kadel** [comment]
Yep, closing this. addressed in #425
